pyquil/api/_qac.py

```diff
@@ -33,7 +33,7 @@ def get_version_info(self) -> dict:
         raise NotImplementedError
 
     @abstractmethod
-    def quil_to_native_quil(self, program: Program) -> Program:
+    def quil_to_native_quil(self, program: Program, protoquil=False) -> Program:
```

Comment:
@ecpeterson how do you feel about forcing these kind of args to be named args with `*`?

Reply:
Currently indifferent. How do *you* feel / why?

Reply:
I don't feel too strongly either way. Inasmuch as it's nice for the skin over an RPC call to feel like the RPC call and for the RPC call to feel like a real call, and because the latter two expect `protoquil` to be a kwarg, maybe it should be enforced as a kwarg in the former too. Not a huge deal, though.

Reply:
I feel like it could be a good idea. Like you say @ecpeterson, using named args gives me more of a feeling that these methods are mostly just wrappers for the underlying RPC call, which I think is how they should feel. Also, from a backwards compatibility perspective, it makes it easier in the future if we want to remove it or rename it. Last, positional boolean args kinda give me the heebiejeebies, but that could just be a me problem.

Reply:
should i also force kwarg for the `QuantumComputer.compile()` method?